tests/fips/test_fips.py

```diff
@@ -0,0 +1,248 @@
+import pytest
+import time
+import logging
+import json
+
+from tests.common import config_reload
+from tests.common.reboot import reboot, REBOOT_TYPE_FAST
+from tests.common.helpers.assertions import pytest_assert as pyassert
+from tests.common.helpers.assertions import pytest_require as pyrequire
+from tests.common.helpers.dut_utils import check_container_state
+from tests.common.helpers.restapi_helper import generate_cert, apply_cert_config, RESTAPI_CONTAINER_NAME
+from helper import reboot_and_wait_for_fips, set_fips_by_patch
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.sanity_check(skip_sanity=True),
+    pytest.mark.disable_loganalyzer,
+    pytest.mark.skip_check_dut_health
+]
+
+
+'''
+This test checks for reset status and sets it
+'''
+
+VERIFY_PYTHON_FIPS = """
+import os, ssl, time
+rand_byte = ssl.RAND_bytes(1)
+time.sleep(1)
+with open(os.path.join("/proc", str(os.getpid()), "maps")) as f:
+  print(f.read())
+"""
+
+def enable_fips(duthost, is_enable=False):
+    enable_value = '0'
+    if is_enable:
+         enable_value = '1'
+    dut_command = "sudo mkdir -p /etc/fips && echo {} | sudo tee /etc/fips/fips_enable".format(enable_value)
+    duthost.shell(dut_command)
+
+def enable_fips_restapi(duthost, is_enable=False):
+    enable_fips(duthost, is_enable)
+    enable_value = '0'
+    if is_enable:
+         enable_value = '1'
+    dut_command = "docker inspect {} | grep /etc/fips/enable_fips || docker exec {} bash -c 'mkdir -p /etc/fips && echo {} > /etc/fips/fips_enable'".format(RESTAPI_CONTAINER_NAME, RESTAPI_CONTAINER_NAME, enable_value)
+    duthost.shell(dut_command)
+
+    # Restart RESTAPI server with the updated config
+    dut_command = "sudo systemctl restart restapi"
+    duthost.shell(dut_command)
+    time.sleep(15)
+
+def enforce_fips(duthost, is_enforce=False):
+    enable_value = '0'
+    if is_enable:
+         enable_value = '1'
+    dut_command = "sudo mkdir -p /etc/fips && echo {} | sudo tee /etc/fips/fips_enable".format(enable_value)
+    duthost.shell(dut_command)
```

Comment:
This function looks exactly the same as `enable_fips`. Can they be combined into one?